sympy/integrals/rde.py

```diff
@@ -414,8 +432,11 @@ def no_cancel_b_large(b, c, n, DE):
     """
     Poly Risch Differential Equation - No cancellation: deg(b) large enough.
 
-    Given a derivation D on k[t], n either an integer or +oo, and b, c
-    in k[t] with b != 0 and either D == d/dt or
+    Explanation
+    ===========
+
+    Given a derivation D on k[t], ``n`` either an integer or +oo, and ``b``,``c``
+    in k[t] with ``b`` != 0 and either D == d/dt or
```

Comment:
Things like this should either be all code or no code. Mixing code and non-code in an equation makes it harder to read.

Reply:
One of the problem is that `D == d/dt`, `k[t]` may not be valid python code, but the documentation had been written in some pseudocodes.
However, for consistency, it can be better to wrap any syntax like that with backticks, even if they may not be valid codes that work with sympy.

Reply:
Changed.

Reply:
Yes, it's pseudocode copied directly from Bronstein's book. It could be rewritten in LaTeX, but these docstrings aren't intended to be included in Sphinx, only for internal use. So I don't know if it makes sense do to that either. IMO using markup in these internal docstrings isn't a big deal, because they aren't included in Sphinx and will only be read in plain text.

Reply:
Then again, it is possible that we may want to include even internal docstrings in Sphinx at some point, so it's probably best to format them consistently with the rest of SymPy.